test/sync-routes.test.js

```diff
@@ -4,29 +4,29 @@ const { test } = require('node:test')
 const Fastify = require('..')
 
 test('sync route', async t => {
-  const fastify = Fastify()
-  t.after(() => fastify.close())
-  fastify.get('/', () => 'hello world')
-  const res = await fastify.inject('/')
+  const app = Fastify()
```

Comment:
Can you revert the `fastify` -> `app` variables change here?

Reply:
Agreed